src/main/java/org/generationcp/middleware/service/api/study/generation/ExperimentDesignService.java

```diff
@@ -0,0 +1,17 @@
+package org.generationcp.middleware.service.api.study.generation;
+
+import com.google.common.base.Optional;
+import org.generationcp.middleware.domain.etl.MeasurementVariable;
+import org.generationcp.middleware.pojos.workbench.CropType;
+import org.generationcp.middleware.service.api.dataset.ObservationUnitRow;
+
+import java.util.List;
+
+public interface ExperimentDesignService {
+
+	void deleteExperimentDesign(int studyId);
```

Comment:
I suggest to rename it by deleteStudyExperimentDesign or deleteExperimentDesignByStudyId. Not clear what the parameter refers to given that in most of the cases when we delete something the received parameter is the id of the object that will be deleted.